TSKit/Public/TSClient.m

```diff
@@ -149,6 +149,47 @@ - (void)disconnect
     self.isSubscribedToChannels = NO;
 }
 
+- (void)wisperConnect:(nullable NSArray<TSUser *>*)users channels:(nullable NSArray<TSChannel *>*)channels {
```

Comment:
The method name is misleading as it does not "connect".  Maybe you can use the underlaying C function name:
```
-(void) requeSetWhisperList(....`
```